conf/app.routes

```diff
@@ -8,4 +8,6 @@ DELETE  /income-tax/nino/:nino/sources/:employmentId/unignore       controllers.
 
 DELETE  /income-tax/nino/:nino/sources/:employmentId/:toRemove      controllers.DeleteEmploymentController.deleteOrIgnoreEmployment(nino: String, employmentId: String, toRemove: String, taxYear: Int)
 
-POST    /income-tax/nino/:nino/sources                              controllers.CreateUpdateEmploymentController.createUpdateEmployment(nino: String, taxYear: Int)
\ No newline at end of file
+POST    /income-tax/nino/:nino/sources                              controllers.CreateUpdateEmploymentController.createUpdateEmployment(nino: String, taxYear: Int)
+
+GET     /income-tax/nino/:nino/other/employment/income              controllers.OtherEmploymentIncomeController.getOtherEmploymentIncome(nino: String, taxYear: Int)
```

Comment:
the backend APIs have `income/other/employments`. Any reason to not leave it the same way? Not that any convinces me, but I am leaning towards keeping it consistent

Reply:
I agree. We can keep it the same as it is in DES. I will incorporate it